src/mongocxx/lib/mongocxx/v_noabi/mongocxx/mongoc_error.hh

```diff
@@ -68,7 +71,13 @@ template <typename exception_type>
     throw exception_type{v_noabi::make_error_code(error), std::move(raw_server_error), error.message};
 }
 
-template <typename exception_type>
+template <
```

Comment:
Curious: was this change intended in this PR? Reverting changes to mongoc_error.h appears to compile locally.